### DISCUSSION TITLE
Map Zoom And Filtering

### DISCUSSION THREAD

**Discussion by WhenPenguinsAttack** [OpenRCT2/OpenRCT2#15695] [Feature requests]
Searched for this in Github and didn't find anything, so decided to post this here.

The minimap in the game should have zoom functions (to make it easier to look at detail and get around big maps, especially after size limits are removed) and filtering functions (so that, say, only stalls show up on the map).  Map filtering would make it much easier in multiplayer and shared map games to determine where other people have placed stalls so we don't accidentally duplicate them and see where stalls are needed much easier (I know about turning off all scenery, but prefer this method).

**Comment by janisozaur** [comment]
Pretty much duplicate of https://github.com/OpenRCT2/OpenRCT2/issues/2782

**Comment by WhenPenguinsAttack** [comment]
Well, the map zoom suggestion is a duplication, but not the map filtering suggestion.
